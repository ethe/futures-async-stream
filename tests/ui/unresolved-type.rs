// SPDX-License-Identifier: Apache-2.0 OR MIT

#![feature(coroutines)]

use futures_async_stream::stream;

#[stream(item = L)] //~ ERROR cannot find type `Left` in this scope
async fn foo() {}

fn main() {}
