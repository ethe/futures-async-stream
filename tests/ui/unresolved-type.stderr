error[E0412]: cannot find type `L` in this scope
 --> tests/ui/unresolved-type.rs:5:17
  |
5 | #[stream(item = L)] //~ ERROR cannot find type `Left` in this scope
  |                 ^ not found in this scope
6 | async fn foo() {}
  |             - help: you might be missing a type parameter: `<L>`
